docs/en/test_and_ci/fuzz_tests.md

```diff
@@ -0,0 +1,68 @@
+# Fuzz Tests
+
+Fuzz tests are a specialized form of [unit tests](../test_and_ci/unit_tests.md) that run code against a large number of random inputs.
+This helps to ensure that the code is robust against any input, not just those expected by the developer.
+
+They can be written like unit tests with possible assertions (`EXPECT_EQ`, etc), and have a set of input parameters.
+The fuzzer then tries to find inputs that cause the code to crash (with enabled Address Sanitizer) or trigger an assertion.
+
+The tests are run as part of normal unit tests, and in a more comprehensive fuzzing mode test.
+For more information see [Running Fuzz Tests](#running-fuzz-tests) below.
+
+## Writing a Fuzz Test
```

Comment:
This is very clear and succinct.

Do you think it worth showing one "real" example to make it a little less abstract? I.e. a code fragment from a real driver test where you are throwing some input at it?

If you don't want to show a worked example, I'd still consider pointing to at least one clear example in code.

Reply:
I think I would point to an example in the code as a real driver needs a fair bit of context.